src/WebJobs.Script/Environment/EnvironmentExtensions.cs

```diff
@@ -156,6 +156,16 @@ public static bool IsLinuxConsumption(this IEnvironment environment)
             return !environment.IsAppService() && !string.IsNullOrEmpty(environment.GetEnvironmentVariable(ContainerName));
         }
 
+        /// <summary>
+        /// Gets a value indicating whether this specific linux consumption container instance is in offline mode.
+        /// </summary>
+        /// <param name="environment">The environment to verify</param>
+        /// <returns><see cref="true"/> if running in a Linux Consumption App Service app and the container is in draining mode; otherwise, false.</returns>
+        public static bool IsLinuxConsumptionContainerDisabled(this IEnvironment environment)
+        {
+            return environment.IsLinuxConsumption() && Utility.IsContainerDisabled();
```

Comment:
Do not add this helper here. Environment extensions should not be using helpers from Utility. Add this check directly in Utility

Reply:
Changed it to this way based on Fabio's earlier comment on this PR.